setup.py

```diff
@@ -36,7 +36,7 @@
 ]
 
 install_requires = [
-    'Flask~=0.0,>=0.12.3',
+    'Flask',
```

Comment:
this might be too flex, why can't we set also `>=1.0.0` or something. so that it wont cover all the other major updates.

Reply:
Because for inspire-next we have 0.12 and for inspirehep we want 1

Reply:
what I meant by "also" is that you can require/allow multiple versions